packages/mui-docs/package.json

```
@@ -29,28 +29,31 @@
     "build:node": "node ../../scripts/build.mjs node",
     "build:stable": "node ../../scripts/build.mjs stable",
     "build:types": "node ../../scripts/buildTypes.mjs",
-    "build:copy-files": "node ../../scripts/copyFiles.mjs ./src/translations/translations.json:./translations/translations.json",
+    "build:copy-files": "node ../../scripts/copyFiles.mjs ./src/translations/translations.json:./translations/translations.json ./src/translations/translations.json:./node/translations/translations.json",
     "prebuild": "rimraf build",
     "release": "pnpm build && pnpm publish",
     "test": "exit 0"
   },
   "dependencies": {
     "@babel/runtime": "^7.23.9",
-    "@mui/base": "workspace:*",
-    "@mui/utils": "workspace:^",
     "clsx": "^2.1.0",
     "nprogress": "^0.2.0",
     "prop-types": "^15.8.1"
   },
   "devDependencies": {
+    "@mui/material": "workspace:*",
+    "@mui/icons-material": "workspace:*",
     "@types/node": "^18.19.25",
     "@types/prop-types": "^15.7.11",
     "@types/react": "^18.2.55",
     "next": "^13.5.1",
     "react": "^18.2.0"
   },
   "peerDependencies": {
-    "@mui/material": "workspace:^",
+    "@mui/material": "^5.0.0",
+    "@mui/base": "*",
+    "@mui/icons-material": "^5.0.0",
+    "@mui/system": "^5.0.0",
     "@types/react": "^17.0.0 || ^18.0.0",
     "next": "^13.5.1",
```

We use Next 14 on MUI X and Base UI
```suggestion
    "next": "^13.5.1 || ^14",
```